src/archivematica/storage_service/storage_service/settings/base.py

```diff
@@ -754,3 +754,5 @@ def _get_email(email):
     )
     INSTALLED_APPS = INSTALLED_APPS + ["django_prometheus"]
     LOGIN_EXEMPT_URLS.append(r"^metrics$")
+
+SILENCED_SYSTEM_CHECKS = environ.get("SS_SILENCED_SYSTEM_CHECKS", "[]")
```

Comment:
If you're expecting the user to set a comma separated list it needs to be converted to a list here:

```suggestion
SILENCED_SYSTEM_CHECKS = environ.get("SS_SILENCED_SYSTEM_CHECKS", "").split(",")
```